web/src/__snapshots__/AlertPane.test.tsx.snap

```diff
@@ -15,14 +15,14 @@ exports[`renders multiple lines of a crash log 1`] = `
         <p>
           Pod crashed
         </p>
-        <p>
+        <time>
```

Comment:
`<time>` inside `<time>` seems weird, as does a `<time>` with no `dateTime` attribute. Can we just get rid of the outer one? Was the `<p>` screwing things up?

Reply:
Good catch! When we added relative time we neglected to remove this wrapping HTML.